src/Mod/Spreadsheet/App/PropertySheet.cpp

```diff
@@ -120,9 +131,14 @@ bool PropertySheet::isValidAlias(const std::string &candidate)
     if (getValueFromAlias(candidate) != 0)
         return false;
 
+    // There is now an unambiguious way of referencing any alias that is a
+    // valid identifier, i.e. prefix it with a '.', which avoids clash
+    // with unit under any circumstances.
+#if 0
```

Comment:
Just a style question: why disable with a preprocessor conditional, rather than commenting it out?

Reply:
I guess you can say it's a personal habit. Because, you know, the original purpose of comment is for, well, comments. I prefer using preprocessor to disable code, and not worry about nested comments and so on.